HotTips/Tips/EDI_CommentCode.md

```diff
@@ -0,0 +1 @@
+﻿Press **Ctrl+K+C** and **Ctrl+K+U** to comment and un-comment code.
```

Comment:
The chords need to be separated by a comma (and a space). Ctrl+K, C  and Ctrl+K, U